tests/integration/security/external_ca/main_test.go

```diff
@@ -92,8 +95,6 @@ components:
       env:
       - name: CERT_SIGNER_DOMAIN
         value: clusterissuers.istio.io
-      - name: EXTERNAL_CA
```

Comment:
Can we keep this?  since we have users depend on this setting already

Reply:
I have updated the docs to align with this: https://github.com/istio/istio.io/pull/12908